pegasus/sites.v3/code.org/views/unsupported_browser.haml

```diff
@@ -5,7 +5,15 @@
 
 :javascript
   $(document).ready(function () {
+    if (!window.isUnsupportedBrowser) {
```

Comment:
One downside to that approach Brad is that in the case where isUnsupportedBrowser is undefined, we won't be able to tell from the callstack in NR whether it's on the initial load or on the delayed load.